tests/data_sources/test_geodataframe_data_source.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+import geopandas as gpd
+import numpy as np
+import pytest
+from pyogrio.errors import DataSourceError
+
+from hydromt.data_sources.geodataframe_data_source import GeoDataFrameDataSource
+
+
+class TestGeoDataFrame:
+    @pytest.fixture(scope="class")
```

Comment:
do want a class scope here? or function? (if that's the correct term) i.e. do we want changes to this to persist for other tests? ( I'd assume no, but am happy to be corrected)

Reply:
Yeah I changed it already